packages/go/dawgs/graph/mocks/graph.go

```diff
@@ -15,20 +15,272 @@
 // SPDX-License-Identifier: Apache-2.0
 
 // Code generated by MockGen. DO NOT EDIT.
-// Source: github.com/specterops/bloodhound/dawgs/graph (interfaces: Batch,Transaction,Database)
+// Source: graph.go
```

Comment:
This new comment seems a lot less helpful. Just wanted to make sure MockGen isn't misconfigured in your environment or something

Edit:
This seems to be on purpose actually so you can see it's from `graph.go`. Is there a way to include the other info or a path too?

Reply:
That comment is autogenerated by mockgen. After looking through the docs, I don't see a way to include additional information without reverting to reflection mode and forcing maintainers to list every declared interface.

As a note, there are definitely trade-offs between generating mocks via source instead of using reflection. We might want to consider pulling this issue into its own discussion and make a PR afterward.